packages/protocol/contracts/ZSC.sol

```diff
@@ -9,7 +9,7 @@ contract ZSC {
     ZetherVerifier zetherverifier;
     BurnVerifier burnverifier;
     uint256 public epochLength; // now in milliseconds.
-
+    uint256 public epochMultiplier; // additional multiplier to take care of different timestamp units in different protocols (ibft = 1, raft = 1000000)
```

Comment:
so the epochLength is in milliseconds, which is 1000000 times of nono-seconds, so the multiplier is correct for Raft (using nano-seconds for block timestamps).

however IBFT timestamps are in seconds, which means the multiplier needs to be `1/1000` instead of `1`